### PR TITLE
Send email when order is confirmed, won't confirm order unless date has been set, updated readme

### PR DESCRIPTION
**Pull request by Rdore88**

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -30,7 +30,9 @@ def destroy
   end
 
   def confirm_order
-    if @order.update(confirmed: true)
+    if !@order.date
+      render json: {status: "failed", message: "Please set a date before you confirm an order"}
+    elsif @order.update(confirmed: true)
```
Comment (IsaacHardy) [inline app/controllers/orders_controller.rb]: Are we sure we need this? I'm not sure if this functionality would be helpful.

Comment (Rdore88) [reply app/controllers/orders_controller.rb]: In the email I made I have something for the date so if the date is empty the user will get a weird empty spot. So we can either have the date in the email and then we would need to make sure there is a date on the order or I can take that out. Which ever works

Comment (IsaacHardy) [reply app/controllers/orders_controller.rb]: I don't think have a date is needed at all.

Comment (Rdore88) [reply app/controllers/orders_controller.rb]: 👍  I will make those changes

**Pull request merged**